File: benchmark_fibonacci.c
#include <stdio.h>

// Fibonacci benchmark - C version
int fibonacci(int n) {
    if (n <= 1) {
        return n;
    }
    return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int result = fibonacci(30);
    printf("Fibonacci(30) completed\n");
    return 0;
}

File: benchmark_fibonacci.rs
// Fibonacci benchmark - Rust version
fn fibonacci(n: i32) -> i32 {
    if n <= 1 {
        return n;
    }
    fibonacci(n - 1) + fibonacci(n - 2)
}

fn main() {
    let result = fibonacci(30);
    println!("Fibonacci(30) completed");
}